tests/runtest.py

```diff
@@ -371,6 +371,40 @@ def check_perf_paranoid(self):
 
         return True
 
+    def is_32bit(self):
+        EI_NIDENT = 16
+
+        # e_ident[] indexes
+        EI_MAG0       = 0
+        EI_MAG1       = 1
+        EI_MAG2       = 2
+        EI_MAG3       = 3
+        EI_CLASS      = 4
+        EI_DATA       = 5
+        EI_VERSION    = 6
+        EI_OSABI      = 7
+        EI_PAD        = 8
```

Comment:
We don't need all of these, just keep `EI_CLASS` here.

Reply:
I removed them.